dsl/src/main/kotlin/com/chutneytesting/kotlin/junit/engine/execution/ExecutionListeners.kt

```diff
@@ -110,23 +111,9 @@ class FileWriterScenarioReportExecutionListener : EnabledTestExecutionListener(e
 
 class SiteGeneratorExecutionListener : EnabledTestExecutionListener(enabledProperty = CONFIG_REPORT_SITE) {
 
-    private var reportRootPathConfig: String? = null
-
-    override fun testPlanExecutionStarted(testPlan: TestPlan) {
-        super.testPlanExecutionStarted(testPlan)
-
-        if (enabled()) {
-            reportRootPathConfig =
-                configurationParameters().get(CONFIG_REPORT_ROOT_PATH.parameter).orElse(CONFIG_REPORT_ROOT_PATH.defaultString())
-        }
-    }
-
     override fun testPlanExecutionFinished(testPlan: TestPlan?) {
         if (enabled()) {
-            ofNullable(reportRootPathConfig).ifPresentOrElse(
-                { SiteGenerator(it).generateSite() },
-                { SiteGenerator().generateSite() }
-            )
+           SiteGenerator().generateSite()
```

Comment:
We have here a regression because this generator doesn't take into account the JUnit properties file.